packages/release-please/test/release-please.ts

```diff
@@ -345,6 +411,38 @@ describe('ReleasePleaseBot', () => {
       assert(executed, 'should have executed the runner');
     });
 
+    it('should try to create a release on an alternate branch', async () => {
```

Comment:
Do we have a test for config [like this](https://github.com/googleapis/nodejs-secret-manager/blob/master/.github/release-please.yml)

```
releaseType: node
```

Where there's a releaser type, but no branch? If not let's add one.

Reply:
Added a test in https://github.com/googleapis/repo-automation-bots/pull/1290/commits/0448967ffbbdc72e8c6692df313d7704c376aba2